src/ngraph/op/experimental/quantized_avg_pool.cpp

```diff
@@ -36,14 +38,20 @@ op::QuantizedAvgPool::QuantizedAvgPool(const shared_ptr<Node>& arg,
 {
     constructor_validate_and_infer_types();
 
-    if (arg->get_element_type() != element::u8 && arg->get_element_type() != element::i8)
+    if (arg.get_element_type() != element::u8 && arg.get_element_type() != element::i8)
```

Comment:
This check was copied and pasted into `validate_and_infer_types()`. Delete it here?